README.md

```diff
@@ -63,6 +63,18 @@ node mynode.lab.local {
 }
 ~~~  
 
+Second basic example, using username/password with single source json file:
+
+~~~puppet
+node mynode.lab.local {
+  class sumo {
+    email                     => 'user@example.com',
+    password                  => 'usersPassword123!', 
+    syncsourceswithsinglejson => 'PathToJsonFile/SingleSources.json', 
```

Comment:
Shouldn't this be a fully qualified path? Or is it relative to a a certain directory?

Reply:
@ghoneycutt : This need to be fully qualified path followed by filename.json. For example: '/etc/source.json'